_posts/2020-08-30-commenting-system-blog.md

```diff
@@ -120,7 +123,7 @@ dotnet restore
 ```
 dotnet run
 ```
-8. Use VS Code or any of your favourite text editor and open the project folder
+8. Use VS Code or your favourite text editor and open the project folder
 9. Notice four new files in addition to the normal files new .NET project area created.
 <p align="center">
```

Comment:
Why is this in HTML not markdown?

Reply:
You mean the image? If yes, because I wanted to center align it. From my experience with markdown, you can only centre align in this way